Makefile

```
@@ -12,25 +12,31 @@ else
 	MOUNT_OPT := 
 endif
 
-build: analyzer deps golang-dependency-provider external-generic yq-external-provider java-external-provider
+build-dir:
+	mkdir -p build
 
-analyzer:
-	go build -o konveyor-analyzer ./cmd/analyzer/main.go
+build: build-dir analyzer deps golang-dependency-provider external-generic yq-external-provider java-external-provider dotnet-external-provider
```

checkmake (0.2.2)

[warning] Target "build" should be declared PHONY.

(phonydeclared)